src/ansys/grantami/bomanalytics/_item_results.py

```diff
@@ -1613,12 +1621,15 @@ class MaterialWithSustainabilityResult(
     MassResultMixin,
     MaterialReferenceWithIdentifiers,
 ):
-    """Describes an individual material included as part of a sustainability query result.
+    """
+    Describes an individual material included as part of a sustainability query result.
```

Comment:
Only flagging as I think it renders differently without the empty line. There are a couple more instances below